### PR TITLE
PRKT-66 Start ScanData at 0 degrees

### PR DESCRIPTION
**Pull request by BryceVichiOJ**
Changed Driver code to start a ScanData set at 0 degrees, rather than 36-72 degrees.

### CODE REVIEW

File: src/parakeet/Driver.cpp [hunk 1]
```diff
@@ -399,6 +383,7 @@ namespace parakeet
         double startAngle = data->from / 10.0;
         double endAngle = (static_cast<double>(data->from) + static_cast<double>(data->span)) / 10.0;
         double anglePerPoint = (endAngle - startAngle) / data->count;
```
Comment (oj-kentd) [inline src/parakeet/Driver.cpp]: Should probably update these to startAngle_deg, etc.

Comment (BryceVichiOJ) [reply src/parakeet/Driver.cpp]: Done.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.2.0] - 2021-06-28
```
Comment (nichmj01) [inline CHANGELOG.md:7]: Since we're don't know the actual release date, typically people would but In-Development until we're about to merge into main.

**Review by nichmj01 — APPROVED** [review]

**Review by oj-kentd — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (BryceVichiOJ) [reply CHANGELOG.md:7]: Good to know. 

I was going to leave out the ChangeLog version bumping until the PR from the sprint branch into dev was done, but I wanted to ensure I was at least tracking changes along the way.

**Pull request merged**